### PR TITLE
[Attendance] skapp 3221 fixes

### PR DESCRIPTION
**Pull request by erandi-rootcode**
…striction

### Summary

-

### How to test

1.

### Project Checklist

- [ ] Changes build without any errors
- [ ] Have written adequate test cases
- [ ] Done developer testing in
  - [ ] Chrome
  - [ ] Firefox
  - [ ] Safari
- [ ] Code is formatted with `npm run format`
- [ ] Code is linted with `npm run check-lint`
- [ ] No unnecessary comments left in code
- [ ] Made corresponding changes to the documentation

#### Other

- [ ] New atomic components added
- [ ] New molecules added
- [ ] New pages(routes) added
- [ ] New dependencies installed

### PR Checklist

- [ ] Pull request is raised from the correct source branch
- [ ] Pull request is raised to the correct destination branch
- [ ] Pull request is raised with correct title
- [ ] Pull request is self reviewed
- [ ] Pull request is self assigned
- [ ] Suitable pull request status labels are added (`ready-for-code-review`)

### Additional Information
-

### CODE REVIEW

**Review by ThinuwanW — COMMENTED** [review]
## 🤖 Claude Code Review

This PR lets privileged users (super admin / attendance admin / attendance manager) act as their own direct-entry target so their own manual time entries bypass the supervisor approval flow. The wiring itself is clean and type-safe, but the new `selfDirectEntryTarget` fallback in `TimesheetDailyRecordTableRow.handleRowActivate` can fire while viewing *another* employee's timesheet (when `targetEmployeeDetails` has not resolved yet), and the my-timesheet header button now enters the direct-entry submit path where a stale `selectedDailyRecord` turns an "add" into an "edit" of an unrelated record. The direct-entry mutations also do not invalidate the queries that back the my-timesheet page, so self entries leave the UI stale.

Found **14** new issue(s): 🔴 5 important, 🟡 5 suggestion(s), 🟣 4 nit(s)

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx [hunk 1]
```diff
@@ -196,6 +199,14 @@ const TimesheetDailyRecordTableRow: FC<Props> = ({
       return;
     }
 
+    if (selfDirectEntryTarget) {
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: 🤖 **Claude** · 🔴 **Important**

The new `selfDirectEntryTarget` branch can hijack another employee's timesheet row. In `IndividualEmployeeTimeReportBody`, `targetEmployeeId` is always passed but `targetEmployeeDetails` comes from `useGetEmployeeById(selectedUser, canDirectlyAddOrEditEntry)` — an async query. While that query is in flight (or if it errors/returns nothing), `targetEmployeeDetails` is `undefined` while `targetEmployeeId` is set, so the first branch is skipped. `isRowInteractive` is still `true` in that window (for an admin, `isManualEntryRestricted` is false, so the `targetEmployeeDetails ? ... : !isManualEntryRestricted` fallback evaluates to true), so the click falls through to this new branch and sets the *acting admin* as the direct-entry target while `selectedDailyRecord` is the *other employee's* record. On submit, `useAddEntry` posts/patches `DIRECT_TIME_ENTRY(currentUser.employeeId)` with the other employee's record data and `recordId` — writing to the wrong person's timesheet. Gate this branch on the absence of an employee context, e.g. `if (!targetEmployeeId && selfDirectEntryTarget) { ... }`, and/or make `isRowInteractive` false whenever `targetEmployeeId` is set but `targetEmployeeDetails` has not loaded.

File: frontend/pages/community/timesheet/my-timesheet.tsx [hunk 2]
```diff
@@ -46,7 +49,7 @@ const MyTimeSheet: NextPage = () => {
       primaryButtonType={ButtonStyle.PRIMARY}
       isPrimaryBtnDisabled={isRestrictionLoading}
       onPrimaryButtonClick={() => {
-        setDirectManualTimeEntryEligibleEmployee(null);
+        setDirectManualTimeEntryEligibleEmployee(selfDirectEntryTarget);
```
Comment (ThinuwanW) [inline frontend/pages/community/timesheet/my-timesheet.tsx]: 🤖 **Claude** · 🔴 **Important**

This is the first code path that sets `directManualTimeEntryEligibleEmployee` without also setting `selectedDailyRecord`. `selectedDailyRecord` is never reset — `employeeTimesheetModalSlice` only ever assigns it, and `EmployeeTimesheetPopupController`'s cleanup effect clears `currentAddTimeChanges` and the direct-entry target but not the selected record. In `useAddEntry.handleTimeEntrySubmit`, the direct-entry branch reads `const existingRecordId = selectedDailyRecord?.timeRecordId || undefined` and calls `editDirectManualTimeEntryMutate` when it is truthy. So: user clicks a daily-log row for Sep 10 (sets `selectedDailyRecord`), closes the modal, then clicks this "Add time entry" header button and picks Sep 12 → the submit PATCHes Sep 10's `recordId` with Sep 12's times, silently overwriting an unrelated record instead of creating a new one. Clear `selectedDailyRecord` (add a setter that accepts `undefined`, or reset it here / in the popup-controller cleanup effect) before opening the ADD_TIME_ENTRY modal, or ignore `selectedDailyRecord.timeRecordId` when `employeeTimesheetModalType === ADD_TIME_ENTRY`.

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx [hunk 3]
```diff
@@ -196,6 +199,14 @@ const TimesheetDailyRecordTableRow: FC<Props> = ({
...
+      if (getTimeEntryModalType(record) === null) return;
+
+      setDirectManualTimeEntryEligibleEmployee(selfDirectEntryTarget);
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: 🤖 **Claude** · 🔴 **Important**

Missing cache invalidation for the newly supported self case. The direct-entry mutations (`useAddDirectTimeEntry` / `useEditDirectTimeEntry` in `src/enterprise/attendance/api/AttendanceApi.ts`) call `invalidateDirectTimeEntryQueries`, which only invalidates `getManagerRecords()`, `getManagerWorkSummary()`, `getEmployeeDailyLogByEmployeeId()` and `getEmployeeRequests()`. The my-timesheet page renders through `useGetDailyLogs` / `useGetEmployeeWorkSummary`, which use `attendanceQueryKeys.getEmployeeDailyLog()` and `getEmployeeWorkSummary()` — neither is invalidated, and neither is `getAttendanceQueryKeys.employeeStatus()`. After a privileged user adds/edits their own entry via the new path, the daily-log table, the worked/break-hours KPI cards and the clock-in widget stay stale until a manual refetch. Add `attendanceQueryKeys.getEmployeeDailyLog()`, `attendanceQueryKeys.getEmployeeWorkSummary()` and `getAttendanceQueryKeys.employeeStatus()` to `invalidateDirectTimeEntryQueries` (or reuse `invalidateAttendanceTimeRecordQueries` alongside it).

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx [hunk 4]
```diff
@@ -196,6 +199,14 @@ const TimesheetDailyRecordTableRow: FC<Props> = ({
       return;
     }
 
+    if (selfDirectEntryTarget) {
+      if (getTimeEntryModalType(record) === null) return;
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: 🤖 **Claude** · 🔴 **Important**

Behavioral regression: the new early-return skips the period-availability check for the user's own rows. Previously a self row click fell through to `mutate()` → `handleAvailability`, which opened `ONGOING_TIME_ENTRY_BY_EDIT` when the user is currently clocked in (`status` is START/PAUSE/RESUME and `isToday(record.date)`) and `TIME_REQUEST_EXISTS_BY_EDIT` when `editTimeRequests`/`manualEntryRequests` exist. With `selfDirectEntryTarget` set, both guards are gone and `handleEdit()` runs immediately. Opening `EDIT_AVAILABLE_TIME_ENTRY` on an in-progress record is the risky one: `AddEditTimeEntry` reads `timeSlots.at(-1).endTime`, which has no value for a running slot, so the clock-out field is populated from an invalid date. The same class of bypass applies to the header "Add time entry" path — in `useAddEntry.handleTimeEntrySubmit` the `directManualTimeEntryEligibleEmployee` branch returns before the `ADD_TIME_ENTRY` branch, so `getModalBeforeManualEntry`'s ONGOING / TIME_ENTRY_EXISTS / leave / holiday confirmations never run for privileged users. Bypassing the *supervisor approval* is the intent of this PR; bypassing the ongoing-session and overlap guards looks unintended — keep the availability check (or at least the ongoing-session check) before entering the direct-entry flow for self.

File: frontend/src/community/attendance/types/timeSheetTypes.ts [hunk 5]
```diff
@@ -226,6 +226,7 @@ export interface DirectEntryEmployeeType {
   employeeId: number;
   employeeName: string;
   date?: string;
+  isSelf?: boolean;
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/types/timeSheetTypes.ts]: 🤖 **Claude** · 🟡 **Suggestion**

`isSelf` is an optional flag on shared store state, and the consumer in `AddEditTimeEntry` relies on `!directManualTimeEntryEligibleEmployee.isSelf` being true for objects that simply omit it (e.g. `EmployeeTimeRecordsTable.tsx:148` and `TimesheetDailyRecordTableRow.tsx:191` construct the target without the flag). That works today but is easy to get wrong as new producers are added. Prefer deriving the condition at the point of use — `directManualTimeEntryEligibleEmployee.employeeId === employeeDetails?.employeeId` — or make the field required so every producer must state it explicitly.

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts [hunk 6]
```diff
@@ -32,10 +41,28 @@ const useManualEntryRestriction = (): ManualEntryRestrictionResult => {
   const canDirectlyAddOrEditEntry =
     !isError && isRestrictionEnabled && canManageTimeEntries;
 
+  // Users who can add or edit entries directly are their own valid direct
+  // entry target, so their own entries bypass the supervisor approval flow.
+  const selfDirectEntryTarget = useMemo<DirectEntryEmployeeType | null>(() => {
+    if (!canDirectlyAddOrEditEntry || !employeeDetails?.employeeId) {
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: 🤖 **Claude** · 🟣 **Nit**

`!employeeDetails?.employeeId` treats a numeric id of `0` as "no employee". Employee ids are almost certainly 1-based so this is unlikely to bite, but `employeeDetails?.employeeId === undefined` (or `== null`) expresses the intent precisely and avoids the falsy-number trap.

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx [hunk 7]
```diff
@@ -196,6 +199,14 @@ const TimesheetDailyRecordTableRow: FC<Props> = ({
...
+
+      setDirectManualTimeEntryEligibleEmployee(selfDirectEntryTarget);
+      handleEdit();
+      return;
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: 🤖 **Claude** · 🟣 **Nit**

`getTimeEntryModalType(record) === null` is now evaluated in both the `targetEmployeeDetails` branch and the new `selfDirectEntryTarget` branch, and again inside `handleEdit`. Hoist it once to the top of `handleRowActivate` (`const modalType = getTimeEntryModalType(record);`) and reuse the result, so the three call sites can't drift apart.

File: frontend/pages/community/timesheet/my-timesheet.tsx (hunk 2 above)
Comment (ThinuwanW) [inline frontend/pages/community/timesheet/my-timesheet.tsx]: 🤖 **Claude** · 🔴 **Important**

Setting `directManualTimeEntryEligibleEmployee` here does more than swap the API call — it makes `useAddEntry.handleTimeEntrySubmit` take its early `if (directManualTimeEntryEligibleEmployee)` branch and return, which skips `submitManualTimeEntry` entirely. That means the whole `getModalBeforeManualEntry` guard set is now bypassed for privileged users on this page: ONGOING_TIME_ENTRY (user is currently clocked in and entering time for today), TIME_REQUEST_EXISTS (a pending edit/manual-entry request already covers the period), TIME_ENTRY_EXISTS (slots already recorded for that window), CONFIRM_TIME_ENTRY (approved/pending leave on that date) and CONFIRM_HOLIDAY_TIME_ENTRY. Before this change `setDirectManualTimeEntryEligibleEmployee(null)` guaranteed the request flow and all five confirmations ran. Note this is separate from the row-click regression already reported: this is the page's primary-button path, where the check happens at submit time in `useAddEntry`, not at click time via `useGetPeriodAvailabilityMutation`. A privileged user can now silently overwrite/duplicate an in-progress or already-recorded day with no warning. Fix: in `handleTimeEntrySubmit`, run `getModalBeforeManualEntry(values, timeAvailability, status)` before the direct-entry branch and, when it returns a non-null modal type, route to that confirmation modal (the confirm modals' "proceed" handlers then need to re-enter the direct-entry call rather than `manualEntryMutate`).

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts [hunk 8]
```diff
@@ -32,10 +41,28 @@ const useManualEntryRestriction = (): ManualEntryRestrictionResult => {
   const canDirectlyAddOrEditEntry =
     !isError && isRestrictionEnabled && canManageTimeEntries;
 
+  // Users who can add or edit entries directly are their own valid direct
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: 🤖 **Claude** · 🟡 **Suggestion**

`canDirectlyAddOrEditEntry` is `isSuperAdmin || isAttendanceAdmin || isAttendanceManager`, so this memo also makes every ATTENDANCE_MANAGER a direct-entry target for themselves. That is a wider policy change than "privileged users whose approval would otherwise be impossible": an attendance manager who does have a supervisor could previously only file a manual-entry request against that supervisor, and now writes their own time records with no approval trail. The community backend's `validateManualEntryRestriction` (TimeServiceImpl.java:2200) only lets these roles *past the restriction* into the request flow — it never grants them self-approval. Please confirm this separation-of-duties widening is intended; if only admins/super-admins should self-approve, gate `selfDirectEntryTarget` on `isSuperAdmin || isAttendanceAdmin` rather than reusing `canDirectlyAddOrEditEntry`, and add a comment explaining why the two predicates differ.

File: frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx [hunk 9]
```diff
@@ -353,15 +353,16 @@ const AddEditTimeEntry = ({ setFromDateTime, setToDateTime }: Props) => {
...
-      {directManualTimeEntryEligibleEmployee && (
-        <InputField
-          label={translateText(["directEntryEmployeeLabel"])}
-          inputName={"direct_entry_employee"}
-          value={directManualTimeEntryEligibleEmployee.employeeName}
-          labelStyles={classes.disabledInputFieldLabel}
-          isDisabled
-        />
-      )}
+      {directManualTimeEntryEligibleEmployee &&
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx]: 🤖 **Claude** · 🟡 **Suggestion**

Only one of the two `directManualTimeEntryEligibleEmployee` consumers in this file was updated for the new `isSelf` flag. The submit button further down still reads `directManualTimeEntryEligibleEmployee ? translateText(["directEntrySaveBtnTxt"]) : translateText(["submitRequestBtnTxt"])`, so a self entry now hides the employee name field (making the modal look like the ordinary self flow) but keeps the direct-entry button copy. Either both should treat `isSelf` as "render as the normal self flow", or neither should — pick one and apply it consistently. If the direct-entry label is deliberate (it is arguably more accurate, since no approval request is created), add a short comment here so the asymmetry does not read as an oversight.

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts [hunk 10]
```diff
@@ -32,10 +41,28 @@ const useManualEntryRestriction = (): ManualEntryRestrictionResult => {
...
+  // entry target, so their own entries bypass the supervisor approval flow.
+  const selfDirectEntryTarget = useMemo<DirectEntryEmployeeType | null>(() => {
+    if (!canDirectlyAddOrEditEntry || !employeeDetails?.employeeId) {
+      return null;
+    }
+
+    return {
+      employeeId: employeeDetails.employeeId,
+      employeeName: concatStrings([
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: 🤖 **Claude** · 🟡 **Suggestion**

`employeeName` is computed here but never rendered: the only consumer of `DirectEntryEmployeeType.employeeName` is the `InputField` in `AddEditTimeEntry`, which this same PR hides whenever `isSelf` is true. So this `concatStrings(...).trim()` is dead work on every recompute. It is also the third copy of the same first/last-name join — `commonUtil.ts` already exports `getEmployeeAvatarName` as `concatStrings([firstName, lastName]).trim()`, and `TimesheetDailyRecordTableRow` line 193 inlines it again. Either reuse `getEmployeeAvatarName({ firstName, lastName })` here (and in the row), or, if the name genuinely is not needed for the self target, reconsider the shape of `DirectEntryEmployeeType` so `employeeName` is not a required field the self case has to fabricate.

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts [hunk 11]
```diff
@@ -1,20 +1,29 @@
+import { useMemo } from "react";
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: 🤖 **Claude** · 🟡 **Suggestion**

📁 **File-level observation**

No test coverage accompanies this change. `useManualEntryRestriction` now owns role + tier + config-dependent business logic that decides whether a user writes time records directly or files an approval request, and it has no test file at all. The two components touched (`TimesheetDailyRecordTable.test.tsx`, `TimesheetDailyRecordTableRow.test.tsx`) are render-only smoke tests that never exercise `handleRowActivate`, so the new `selfDirectEntryTarget` branch is completely untested. Please add: (a) a hook test covering `selfDirectEntryTarget` being null when the restriction is disabled, null for a non-privileged employee, and populated for each privileged role; and (b) a row test asserting that clicking a self row with a `selfDirectEntryTarget` opens the edit modal and sets the store target, while a row without one still calls the period-availability mutation.

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx [hunk 12]
```diff
@@ -52,6 +53,7 @@ interface Props {
   targetEmployeeDetails?: L1EmployeeType;
   isRowInteractive: boolean;
   isManualEntryRestricted: boolean;
+  selfDirectEntryTarget?: DirectEntryEmployeeType | null;
```
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: 🤖 **Claude** · 🟣 **Nit**

`selfDirectEntryTarget?: DirectEntryEmployeeType | null` is both optional and nullable, which makes "caller forgot to pass it" indistinguishable from "there is no self target". Because omitting it silently falls through to the old request flow, a future component that renders this row without the prop would quietly regress privileged users back to approval requests with no type error. Drop the `?` and declare it `selfDirectEntryTarget: DirectEntryEmployeeType | null` so every call site must decide explicitly (the only current caller, `TimesheetDailyRecordTable`, already always passes it).

File: frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx (hunk 9 above)
Comment (ThinuwanW) [inline frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx]: 🤖 **Claude** · 🟣 **Nit**

📁 **File-level observation**

With a self target, the modal type on the my-timesheet path is `ADD_TIME_ENTRY`, which is the one branch of `isSubmitDisabled` that returns `true` while `isGetTimeAvailabilityLoading` is set. So the `useGetPeriodAvailability` effect still refires on every date/from/to change and the Save button is disabled waiting on a response that the direct-entry branch in `handleTimeEntrySubmit` then throws away. This combination could not occur before, because the pre-existing target-employee direct flow only ever uses modal types produced by `getTimeEntryModalType` (never `ADD_TIME_ENTRY`). Consider skipping the availability fetch and its submit gating when `directManualTimeEntryEligibleEmployee` is set — or, if the fix for the missing confirmation guards is applied, keep the fetch and actually use its result.

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts (hunk 8 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: Not Valid

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx (hunk 1 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: Not Applicable Anymore

File: frontend/pages/community/timesheet/my-timesheet.tsx (hunk 2 above)
Comment (erandi-rootcode) [reply frontend/pages/community/timesheet/my-timesheet.tsx]: Not Applicable Anymore

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx (hunk 3 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: Not Applicable Anymore

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx (hunk 4 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: Not Applicable Anymore

File: frontend/src/community/attendance/types/timeSheetTypes.ts (hunk 5 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/types/timeSheetTypes.ts]: Not Applicable Anymore

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts (hunk 6 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: Not Applicable Anymore

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx (hunk 7 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: Not Applicable Anymore

File: frontend/pages/community/timesheet/my-timesheet.tsx (hunk 2 above)
Comment (erandi-rootcode) [reply frontend/pages/community/timesheet/my-timesheet.tsx]: Not Applicable Anymore

File: frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx (hunk 9 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/AttendanceModals/AddEditTimeEntry/AddEditTimeEntry.tsx]: Not Applicable Anymore

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts (hunk 10 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: Not Applicable Anymore

File: frontend/src/community/attendance/hooks/useManualEntryRestriction.ts (hunk 11 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/hooks/useManualEntryRestriction.ts]: Not Applicable Anymore

File: frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx (hunk 12 above)
Comment (erandi-rootcode) [reply frontend/src/community/attendance/components/molecules/TimesheetDailyRecordTableRow/TimesheetDailyRecordTableRow.tsx]: Not Applicable Anymore

**Review by akilarootcode — APPROVED** [review]

**Review by ThinuwanW — APPROVED** [review]
